inc/search.class.php

```diff
@@ -1332,9 +1364,11 @@ static function constructData(array &$data, $onlycount = false) {
          // Get rows
 
          // if real search seek to begin of items to display (because of complete search)
-         if (!$data['search']['no_search']) {
+         // FIXME: does not seems possible with PDO... A data_seek is done withing the fecth*
+         // maybe replace $DBread->fetchAssoc with $DBread->result in the above while loop?
+         /*if (!$data['search']['no_search']) {
             $DBread->data_seek($result, $data['search']['start']);
```

Comment:
It is possible, but you have to:
1. Specify that your PDO cursor is scrollable while preparing the request: `$db->prepare($sql, [\PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL]);`. If you do not do that, the cursor will not be able to go backward.
2. Use the `PDOStatement::fetch()` method like this: `$result->fetch(\PDO::FETCH_ASSOC, \PDO::FETCH_ORI_ABS, $data['search']['start']);`

Reply:
I saw that, but it seems this does not really work as expected... I wonder anyways if that can not be achieved in another way (not sure this is still relevant, search seems to be working).